### ISSUE TITLE
Questions about Project and choice of Tech

### ISSUE THREAD

**Issue by Correct-Syntax** [easylogic/editor#110]
Hi @easylogic,

This is a pretty neat project and idea. :)

I had a few questions about the project and your choice of technology for this project.

**1.** Is your **goal** here to basically _a website-builder_ or is this to _ease the load of work involved in creating a website_ or is this mainly _a designer's tool_?
**2.** Are you happy with your choice of using pure Javascript and (it seems) your own custom-made libraries for it? **Is there any reason you didn't use Vue or React or even Electron?** (Please note: I am not questioning your choice of tech so much as I am wondering if the route you've gone as far as creating an application like this is the way to go or not...)

Thank you for your time!

**Comment by easylogic** [comment]
hi @Correct-Syntax  

Thanks for your interest.

**To answer the first question,**

First of all, let me tell you a little about what made me start this project.

I am currently working as a frontend developer (but making design tools is not my main job).

Usually, when creating a campaign site, there are many ways that designers design with a design tool and then developers re-code. At this time, there are areas where trivial numbers and expressions are blocked. So, the design result cannot be used as it is, and developers have to work on it again. I started a project to close the gap between such designers and developers.

Currently, I think the most similar projects are UXPin or Framer.

However, the goal is to be a design tool that represents the quality of Figma.
The reason is that designers mainly create content, so I thought that if the tools that designers use most can be reproduced on the web as they are, it will be useful.

**To answer the second question,**

I am currently the maintainer of the open source WYSIWYG editor called summernote. 

With my experiences at that time, I researched a lot of ways to quickly and easily create UI, and I actually use it as the editor's main technology. At this time, I learned a lesson. Currently the biggest requirement in summernote is to drop the dependencies of jquery and bootstrap. Bootstrap was structured so that it was not used, but jquery was the basis of the editor, so it cannot be changed. 

Major skills are changing over time. It's good to adopt some specific technology, but I think it's good to have your own technology to respond to everything. 

So it's not bad to leave the current editor as it is and make React and Vue wrappers. However, due to the nature of the editor, there are many issues that need to be handled to the low level, so I am trying to solve it with  myself own technology.

In the case of electron, I tried wrapping it as a test. (It can also be used as an extension in chrome webstore.) Since it is being developed by myself, it is not easy to manage various functions, so it is not currently in progress.

I don't know when it will be, but I think it's good to configure the electron version as well when the cloud service is available in the future.

---

The biggest concern at the moment is that it is not a cloud-type service, so it is not possible to organize how to manage large resources in the form of blobs.

Thanks for reading.
